DeviceBridgeE2E/src/device-bridge.spec.ts

```diff
@@ -0,0 +1,205 @@
+import anyTest, { TestInterface } from 'ava';
```

Comment:
Can you add test cases for:
- C2D message subscription
- Getting the current connection status (not the subscription, but the point-in-time one)